app/ui-react/syndesis/src/index.tsx

```diff
@@ -1,13 +1,14 @@
+import './index.css';
+// tslint:disable-next-line:ordered-imports
+// Don't move this line, otherwise we anger the pf4
```

Comment:
you moved that line, is that ok?  :smile: 

Wonder do we need this comment still?

Reply:
👍 yea I think it can be removed now